scale_build/config.py

```diff
@@ -47,6 +47,7 @@ def get_normalized_value(value, _type, default_value=None):
 TRUENAS_BRANCH_OVERRIDE = get_env_variable('TRUENAS_BRANCH_OVERRIDE', str)
 TRY_BRANCH_OVERRIDE = get_env_variable('TRY_BRANCH_OVERRIDE', str)
 VERSION = get_env_variable('TRUENAS_VERSION', str, f'{_VERS}-{BUILD_TIME_OBJ.strftime("%Y%m%d-%H%M%S")}')
+TRUENAS_VENDOR = get_env_variable('TRUENAS_VENDOR', str)
```

Comment:
Should we default to some value here ?
Motivation is that i think we should be consistent that the vendor file exists at all times

Reply:
No, the vendor file should _NOT_ exist at all times. It only exists when someone builds an ISO image specifying this environment variable.